js/src/ui/Dropdown/dropdown.js

```diff
@@ -0,0 +1,83 @@
+/* This file is part of Parity.
+/*
+/* Parity is free software: you can redistribute it and/or modify
+/* it under the terms of the GNU General Public License as published by
+/* the Free Software Foundation, either version 3 of the License, or
+/* (at your option) any later version.
+/*
+/* Parity is distributed in the hope that it will be useful,
+/* but WITHOUT ANY WARRANTY; without even the implied warranty of
+/* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/* GNU General Public License for more details.
+/*
+/* You should have received a copy of the GNU General Public License
+/* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/** USAGE:
+  Options = [
+    {
+      text: 'Jenny Hess',
+      value: 'Jenny Hess',
+      image: { avatar: true, src: '/assets/images/avatar/small/jenny.jpg' },
+    },
+   ...
+  ]
+
+  <Dropdown placeholder='Select Friend' fluid selection options={Options} />
+**/
+
+import React, { PropTypes } from 'react';
+import { Dropdown as DropdownUI } from 'semantic-ui-react';
+
+import styles from './dropdown.css';
+
+export default function Dropdown ({ defaultValue, disabled, fluid, icon, name, onBlur, onChange, onClick, onClose, onFocus, options, placeholder, scrolling, search, selection, text, value }) {
+  return (
+    <DropdownUI
+      className={ styles.dropdown }
+      defaultValue={ defaultValue }
+      disabled={ disabled }
+      fluid={ fluid }
+      icon={ icon }
+      name={ name }
+      onBlur={ onBlur }
+      onChange={ onChange }
+      onClick={ onClick }
+      onClose={ onClose }
+      onFocus={ onFocus }
+      options={ options }
+      placeholder={ placeholder }
+      scrolling={ scrolling }
+      search={ search }
+      selection={ selection }
+      text={ text }
+      value={ value }
+    />
+  );
+}
+
+Dropdown.propTypes = {
+  defaultValue: PropTypes.number,    // Initial value via index.
+  disabled: PropTypes.bool,          // A disabled dropdown menu or item does not allow user interaction.
+  fluid: PropTypes.bool,             // A dropdown can take the full width of its parent.
+  icon: PropTypes.any,               // Shorthand for Icon.
+  name: PropTypes.func,              // Name of the hidden input which holds the value.
+  onBlur: PropTypes.func,            // Called on blur.
+  onChange: PropTypes.func,          // Called when the user attempts to change the value.
+  onClick: PropTypes.func,           // Called on click.
+  onClose: PropTypes.func,           // Called on close.
+  onFocus: PropTypes.func,           // Called on focus.
+  options: PropTypes.any,            // Array of Dropdown.Item props e.g. `{ text: '', value: '' }`
+  placeholder: PropTypes.string,     // Placeholder text.
+  scrolling: PropTypes.bool,         // A dropdown can have its menu scroll.
+  search: PropTypes.bool,            // A selection dropdown can allow a user to search through a large list of choices.
```

Comment:
Is this used? (Along with next property?)

Looking at the current actual usage, prefer fewer options (what is needed) than the kitchen-sink. Having sane defaults just makes things simpler, i.e. we now pass `fluid` everywhere it seems. Same with selection, etc.